### PR TITLE
BULK VALIDATION description-missing

### PR DESCRIPTION
**Pull request by mSarts**
## Summary

Fixes 30 issues across 30 files.

Fixes no issue numbers.

### CODE REVIEW

File: dotnet-desktop-guide/framework/winforms/controls/how-to-show-a-font-list-with-the-fontdialog-component.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "How to: Show a Font List with the FontDialog Component"
+description: Learn how to show a font list with the FontDialog component, which allows users to select a font, as well as change its display aspects, such as its weight and size.
```
Comment (brittmsantos) [inline dotnet-desktop-guide/framework/winforms/controls/how-to-show-a-font-list-with-the-fontdialog-component.md:3]: This one's over the 160 character limit. I'd shorten it to: "Learn how to show a font list with the FontDialog component, which allows users to select a font, as well as change its display aspects."

File: dotnet-desktop-guide/framework/winforms/controls/how-to-show-a-color-palette-with-the-colordialog-component.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "How to: Show a Color Palette with the ColorDialog Component"
+description: Learn how to show a color palette with the ColorDialog component, which displays a palette of colors and returns a property containing the color the user has selected.
```
Comment (brittmsantos) [inline dotnet-desktop-guide/framework/winforms/controls/how-to-show-a-color-palette-with-the-colordialog-component.md:3]: This one's over the 160 character limit. I'd shorten it slightly to: "Learn how to show a color palette with the ColorDialog component, which displays a palette of colors and returns a property containing the user-selected color."

File: dotnet-desktop-guide/framework/winforms/controls/how-to-show-a-font-list-with-the-fontdialog-component.md (hunk 1 above)
Comment (mSarts) [reply dotnet-desktop-guide/framework/winforms/controls/how-to-show-a-font-list-with-the-fontdialog-component.md]: Addressed.

File: dotnet-desktop-guide/framework/winforms/controls/how-to-show-a-color-palette-with-the-colordialog-component.md (hunk 2 above)
Comment (mSarts) [reply dotnet-desktop-guide/framework/winforms/controls/how-to-show-a-color-palette-with-the-colordialog-component.md]: Addressed.

**Comment by mSarts** [comment]
**Global effort to fix validation errors**

 @adegeo   

The Content & Learning team is fixing build validation errors on docs.microsoft.com. This effort will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. I’d very much appreciate your timely review and feedback on these changes I’ve suggested. Thanks!

**Comment by mSarts** [comment]
@adegeo - I'm following up on this PR. Have you had a chance to review? cc: @v-emilypr

**Review by adegeo — APPROVED** [review]

**Pull request merged**